### PR TITLE
Update documentation, promote using modern Gradle APIs. Remove Travis configuration

### PR DESCRIPTION
**Pull request by mateuszkwiecinski**
Fixes https://github.com/jeremymailen/kotlinter-gradle/issues/195

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Kotlinter Gradle
 
-[![Build Status](https://api.travis-ci.org/jeremymailen/kotlinter-gradle.svg?branch=master)](https://travis-ci.org/jeremymailen/kotlinter-gradle)
+[![Build Status](https://github.com/jeremymailen/kotlinter-gradle/workflows/Build%20Project/badge.svg)](https://github.com/jeremymailen/kotlinter-gradle/actions)
+[![Latest Version](https://img.shields.io/maven-metadata/v/https/plugins.gradle.org/m2/org/jmailen/gradle/kotlinter-gradle/maven-metadata.xml?label=gradle)](https://plugins.gradle.org/plugin/org.jmailen.kotlinter)
```
Comment (mateuszkwiecinski) [inline README.md:4]: I updated old badge and added new one, with the latest published artifact version on Gradle Plugin Portal
image 
(the "gradle" label can be set to whatever you want. Usually I saw there `jCenter` or `mavenCentral`, but `Gradle Plugin Portal` seemed too _lengthy_ :P)

The old badge was outdated, right? All CI pipelines have already been migrated to Github Actions, haven't they? (based on that assumption I removed the `.travis.yml` file

File: README.md [hunk 2]
```diff
@@ -144,8 +135,8 @@ tasks.check {
 <summary>Groovy</summary>
 
 ```groovy
-check {
-    dependsOn "installKotlinterPrePushHook"
+tasks.named('check') {
+    dependsOn 'installKotlinterPrePushHook'
```
Comment (mateuszkwiecinski) [inline README.md:139]: I committed to `'` for Groovy scripts, as that seemed to be the convention here. I personally prefer `"` as they work both in Groovy and in Kotlin DSL scripts. It feels more practical, but didn't want to change too much.

If the old single quote isn't a hard requirement, I can easily update them all.

File: README.md [hunk 3]
```diff
@@ -164,7 +155,7 @@ kotlinter {
     indentSize = 4
     reporters = arrayOf("checkstyle", "plain")
     experimentalRules = false
-    disabledRules = emptyArray<String>()
+    disabledRules = emptyArray()
```
Comment (mateuszkwiecinski) [inline README.md:158]: It seems like passing the type isn't needed anymore. When I tried that on my own, the type was greyed out

File: README.md [hunk 4]
```diff
@@ -249,19 +236,21 @@ If you aren't using autoconfiguration from a supported plugin or otherwise need
...
+        )
     )
 }
 
-tasks.create<FormatTask>("ktFormat") {
+tasks.register<FormatTask>("ktFormat") {
     group = "formatting"
     source(files("src"))
-    report = file("build/format-report.txt")
+    report.set(file("build/format-report.txt"))
```
Comment (mateuszkwiecinski) [inline README.md:253]: Report is a `RegularFileProperty` and can't be reassigned, there is no `setReport` method (like in `group =` case above)

File: README.md (hunk 1 above)
Comment (jeremymailen) [reply README.md:4]: Yes, thank you!

**Review by jeremymailen — APPROVED** [review]
Terrific, great cleanups @mateuszkwiecinski

**Pull request merged**